src/app/services/soum-data.service.js

```diff
@@ -0,0 +1,108 @@
+/* ngInject */
+export function SoumData($log, $http, $q, Config) {
+  const username = 'adbmongolia';
+
+  return {
+    load: soumId => {
```

Comment:
You shouldn't need this wrapper promise here. This should be functionally equivalent to:
```
return loadSoumData(soumId).then(responses => {
   // handle transformations
  return soum;
});